addons/website/static/tests/tours/html_editor.js

```diff
@@ -229,6 +233,7 @@ tour.register('test_html_editor_scss', {
             extra_trigger: `body:not(:has(div.ace_line:contains("${adminCssModif}")))`,
             trigger: `body:not(:has(div.ace_line:contains("${demoCssModif}")))`,
             run: function () {}, // it's a check
+            timeout: 30000,
```

Comment:
Very quick comment line would be cool IMHO, even if the info is on the commit message, to quickly get an idea of why this is needed